cli/tauri.js/templates/src-tauri/src/build.rs

```diff
@@ -3,10 +3,13 @@ extern crate winres;
 
 #[cfg(windows)]
 fn main() {
-  let mut res = winres::WindowsResource::new();
-  res.set_icon("icons/icon.ico");
-  res.compile().expect("Unable to find visual studio tools");
-}
+  if std::path::Path::new("icons/icon.ico").exists() {
```

Comment:
This hardcodes the `icons/icon.ico` location. We are specifying its location in the `src-tauri/Cargo.toml` file.

Reply:
I know, I am working on that.

Reply:
hmm. wouldn't it be better to do that in this loop:
`cli/tauri-cli/src/bundle/settings.rs`
```
  /// Returns an iterator over the icon files to be used for this bundle.
  pub fn icon_files(&self) -> ResourcePaths<'_> {
    match self.bundle_settings.icon {
      Some(ref paths) => ResourcePaths::new(paths.as_slice(), false),
      None => ResourcePaths::new(&[], false),
    }
  }
```

Reply:
Not that function but yes, in the bundler is where this will go.